### DISCUSSION TITLE
Welcome to CodeGeneration Discussions!

### DISCUSSION THREAD

**Discussion by SemPlaatsman** [SemPlaatsman/CodeGeneration#62] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community and solve problems. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LuukBakkum** [comment]
User heeft geen iban dus hoe zoek ik naar account/{iban}. neem aan dat het dan in twee keer moet of nie? want bij transaction staat username welke in kan gebruiken om de iban van de user te pakken.

**Reply by LuukBakkum** [reply]
Misschien een extra endpoint search by userId? toevoegen
